.github/ISSUE_TEMPLATE

```diff
@@ -3,14 +3,32 @@ or visit: https://gitter.im/PHP-CS-Fixer
 
 When reporting an issue (bug) please provide the following information:
 
-The PHP version you are using:
-$ php -v
+The PHP version you are using (`$ php -v`):
+=> ....................................
 
-PHP CS Fixer version you are using:
-$ php-cs-fixer -V
+PHP CS Fixer version you are using (`$ php-cs-fixer -V`):
+=> ....................................
 
-The command you use to run the fixer (and the configuration file you are using if any).
+The command you use to run PHP CS Fixer:
+=> ....................................
 
-A minimal sample of valid PHP code that is not fixed correctly (if applicable).
+The configuration file you are using, if any:
+```php
+=> ....................................
+```
 
-The fixed version of that code after you run the fixer and the version you expected.
+Minimal samples of PHP code, if applicable:
```

Comment:
Please add sth like `Please post code as text (using proper markup). Do not post screenshots of code.`